charts/codesealer/templates/configmap-modsec.yaml

```diff
@@ -4,23 +4,56 @@ metadata:
   name: modsec-conf
   labels:
    {{- include "codesealer.labels" . | nindent 4 }}
+  {{- if .Values.worker.config.bootloader.fsEndpoints }}
   namespace: {{ .Values.worker.ingress.namespace }}
+  {{ else }}
+  namespace: {{ .Values.ingress.namespace }}
+  {{- end }}
 data:
   modsec.conf: |
     Include modsecurity/modsecurity.conf
 
     # OWASP CRS v3 rules
     Include modsecurity/crs-setup.conf
     {{- if .Values.worker.config.endpoint.rules.rules900.enabled }}
+
     Include modsecurity/rules/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf
     {{ else }}
+
     # Include modsecurity/rules/REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf
     {{- end }}
     {{- if .Values.worker.config.endpoint.rules.rules901.enabled }}
+
     Include modsecurity/rules/REQUEST-901-INITIALIZATION.conf
     {{ else }}
+    
     # Include modsecurity/rules/REQUEST-901-INITIALIZATION.conf
     {{- end }}
+    {{- if .Values.worker.config.endpoint.rules.rules903.enabled }}
+    Include modsecurity/rules/REQUEST-903.9001-DRUPAL-EXCLUSION-RULES.conf
+
+    Include modsecurity/rules/REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf
+    
+    Include modsecurity/rules/REQUEST-903.9003-NEXTCLOUD-EXCLUSION-RULES.conf
+    
+    Include modsecurity/rules/REQUEST-903.9004-DOKUWIKI-EXCLUSION-RULES.conf
+    
+    Include modsecurity/rules/REQUEST-903.9005-CPANEL-EXCLUSION-RULES.conf
+    
+    Include modsecurity/rules/REQUEST-903.9006-XENFORO-EXCLUSION-RULES.conf
+    {{ else }}
+    # Include modsecurity/rules/REQUEST-903.9001-DRUPAL-EXCLUSION-RULES.conf
+    
+    # Include modsecurity/rules/REQUEST-903.9002-WORDPRESS-EXCLUSION-RULES.conf
+    
+    # Include modsecurity/rules/REQUEST-903.9003-NEXTCLOUD-EXCLUSION-RULES.conf
+    
+    # Include modsecurity/rules/REQUEST-903.9004-DOKUWIKI-EXCLUSION-RULES.conf
+    
+    # Include modsecurity/rules/REQUEST-903.9005-CPANEL-EXCLUSION-RULES.conf
+    
+    # Include modsecurity/rules/REQUEST-903.9006-XENFORO-EXCLUSION-RULES.conf
```

Comment:
Couldn't we just omit these in this case? Same comment elsewhere in this file.

Reply:
The original file had commented rules.  I left for consistency but can remove.